src/roles/foreman/tasks/main.yaml

```diff
@@ -158,6 +158,79 @@
     - worker
     - worker-hosts-queue
 
+- name: Define templated Quadlet for recurring Foreman rake tasks
+  when: foreman_recurring_tasks_enabled | default(true)
+  containers.podman.podman_container:
+    name: "foreman-recurring-%i"
+    quadlet_filename: "foreman-recurring@"
+    state: quadlet
+    image: "{{ foreman_container_image }}:{{ foreman_container_tag }}"
+    sdnotify: false
+    network: host
+    hostname: "{{ ansible_facts['fqdn'] }}"
+    user: foreman
+    working_dir: /usr/share/foreman
+    command: "bash -lc 'foreman-rake ${RAKE_TASK}'"
+    volume:
+      - 'foreman-data-run:/var/run/foreman:z'
+    secrets:
+      - 'foreman-database-url,type=env,target=DATABASE_URL'
+      - 'foreman-seed-admin-user,type=env,target=SEED_ADMIN_USER'
+      - 'foreman-seed-admin-password,type=env,target=SEED_ADMIN_PASSWORD'
+      - 'foreman-settings-yaml,type=mount,target=/etc/foreman/settings.yaml'
+      - 'foreman-katello-yaml,type=mount,target=/etc/foreman/plugins/katello.yaml'
+      - 'foreman-ca-cert,type=mount,target=/etc/foreman/katello-default-ca.crt'
+      - 'foreman-client-cert,type=mount,target=/etc/foreman/client_cert.pem'
+      - 'foreman-client-key,type=mount,target=/etc/foreman/client_key.pem'
+    quadlet_options:
+      - |
+        [Install]
+        WantedBy=default.target foreman.target
+        [Unit]
+        PartOf=foreman.target
+        Requires=foreman.service
+        After=foreman.service
+        StartLimitIntervalSec=0
+      - |
+        [Service]
+        Environment=RAILS_ENV=production
+        EnvironmentFile=/etc/sysconfig/foreman-recurring@%i
+        ExecStartPre=/usr/bin/flock -n /run/foreman-recurring-%i.lock -c /usr/bin/true
```

Comment:
What does this do? `/usr/bin/true` should finish rather quickly, so locking based on that seems odd?